diff --git a/node_modules/oauth4webapi/build/index.js b/node_modules/oauth4webapi/build/index.js
index 4a178ba..a47af07 100644
--- a/node_modules/oauth4webapi/build/index.js
+++ b/node_modules/oauth4webapi/build/index.js
@@ -963,13 +963,13 @@ async function processGenericAccessTokenResponse(as, client, response, ignoreIdT
     if (!validateString(json.access_token)) {
         throw new OPE('"response" body "access_token" property must be a non-empty string');
     }
-    if (!validateString(json.token_type)) {
-        throw new OPE('"response" body "token_type" property must be a non-empty string');
-    }
-    json.token_type = json.token_type.toLowerCase();
-    if (json.token_type !== 'dpop' && json.token_type !== 'bearer') {
-        throw new UnsupportedOperationError('unsupported `token_type` value');
-    }
+    // if (!validateString(json.token_type)) {
+    //     throw new OPE('"response" body "token_type" property must be a non-empty string');
+    // }
+    // json.token_type = json.token_type.toLowerCase();
+    // if (json.token_type !== 'dpop' && json.token_type !== 'bearer') {
+    //     throw new UnsupportedOperationError('unsupported `token_type` value');
+    // }
     if (json.expires_in !== undefined &&
         (typeof json.expires_in !== 'number' || json.expires_in <= 0)) {
         throw new OPE('"response" body "expires_in" property must be a positive number');
